tests/tap.sh

```diff
@@ -175,6 +180,15 @@ for ((i=0;i<${#backends[@]};++i)); do
         continue
     fi
 
+    # grep to skip t506 for MAGMA, range of basis kernels limited for now
+    if [[ "$backend" = *"magma" ]] \
+            && [[ "$1" = "t506"* ]] ; then
+        printf "ok $i0 # SKIP - backend basis kernel not available $1 $backend\n"
+        printf "ok $i1 # SKIP - backend basis kernel not available $1 $backend stdout\n"
+        printf "ok $i2 # SKIP - backend basis kernel not available $1 $backend stderr\n"
+        continue
+    fi
+
```

Comment:
I think this should be in master already?

Reply:
Yes, I didn't change this part. This change comes from merging with the master branch

Reply:
Weird. In that case this branch should know that this doesn't belong to this branch and is already in master.

If I understood the discussion above and the plan is to merge all of these commits into master instead of squashing, then we might want to remove that merging of master into this branch and instead do a rebase.

Reply:
We don't need to merge all commits in this PR, but yes, I would retain at least a few, including ([a0c2d49](https://github.com/CEED/libCEED/pull/516/commits/a0c2d496e17e245ba9197a78c69f3b0c81057b90)) for future reference. When I attempt complex rebases with merges with master in between, git always hates me :( Perhaps someone else can help me with that when we are to merge this branch and at least retain a few commits, including that important one `a0c2d49` at the very least

Reply:
That type of rebase is fundamentally hard. It's better to clean up the history before merging from 'master' (or rebasing your clean branch onto 'master'). When I have a dirty branch with merges, I usually make a squashed merge (yields a single commit on top of 'master'), then use `git citool` to extract out the parts I want in separate commits.

Reply:
Yes, I understand. I had cleaned up the history before the merge with master a little bit, but not extensively to one commit only, to help reviewers follow the changes since the review was still in process.